bootstrap.js

```diff
@@ -230,8 +230,6 @@ function startup(aData, aReason) {
 function shutdown(aData, aReason) {
   // No need to do extra work here
   Log.debug("shutdown, aReason=", aReason);
-  // Load imports for the BOOTSTRAP_REASONS constants; should have happened before in most cases
-  loadImports();
```

Comment:
The thing is the BOOTSTRAP_REASONS are defined in `Cu.import("resource://conversations/modules/config.js", global);` which isn't loaded in loadImports(). Thatswhy I thought it's useless anyway. Now it seems as it was broken before - I'll try to test that case and import the right thing if necessary.